src/components/availability-label/availability-label.tsx

```diff
@@ -39,6 +40,16 @@ export const AvailabilityLabel: React.FC<AvailabilityLabelProps> = ({
 
   const isAvailable = data?.some((item) => item.available);
   const availabilityText = isAvailable ? t("available") : t("unavailable");
+
+  // Track material availability (status) if the button is active - also meaning
+  // it is displayed on the material page
+  if (selected) {
+    track("click", {
+      id: 38,
+      name: "Materiale Status",
+      trackedData: availabilityText
+    });
+  }
```

Comment:
Please consider how this handles rerenders.

A - sometimes involuntary - rerender of a selected availability label will cause a new tracking request. That does not seem like a good idea.

This issue may also be related to our currently high number of availability labels per work.

Reply:
This should be fixed now - I introduced a useEffect() here.